deployments/launcher/pom.xml

```diff
@@ -196,6 +196,11 @@
       <artifactId>swagger-annotations</artifactId>
       <scope>runtime</scope>
     </dependency>
+    <dependency>
+      <groupId>org.jboss.byteman</groupId>
+      <artifactId>byteman</artifactId>
+      <scope>runtime</scope>
+    </dependency>
```

Comment:
I'm a little confused here. As byteman is more like the UT & IT lib, why should we add it as runtime lib?

Reply:
@ligangty It is to provide a [feature](https://issues.redhat.com/browse/ISPN-12825) that is not currently supported in Infinispan. I will send a private message.